### PR TITLE
toQueueShares added

### PR DESCRIPTION
**Pull request by drew-boy**
fix of https://github.com/blend-capital/blend-ui/issues/139

### CODE REVIEW

**Review by mootz12 — APPROVED** [review]
Made changes directly to the branch.

Adding @Ryang-21 as an additional reviewer

File: src/components/backstop/BackstopQueueMod.tsx [hunk 1]
```diff
@@ -55,16 +55,18 @@ export const BackstopQueueMod: React.FC<PoolComponentProps> = ({ poolId }) => {
...
         {poolBackstopUserData.q4w
           .sort((a, b) => Number(a.exp) - Number(b.exp))
-          .map((q4w) => (
+          .map((q4w, index) => (
             <BackstopQueueItem
               key={Number(q4w.exp)}
               poolId={poolId}
               q4w={q4w}
               inTokens={toTokens(q4w.amount)}
+              first={poolBackstopUserData.unlockedQ4W == BigInt(0) && index == 0}
```
Comment (Ryang-21) [inline src/components/backstop/BackstopQueueMod.tsx:69]: Why is this needed?

Comment (mootz12) [reply src/components/backstop/BackstopQueueMod.tsx:69]: Technically anything that is unlocked is still on the Q4W queue. In the UI, we separate these out into "unlocked shares" and a Q4W array of locked entries.

Thus, we only mark the first Q4W entry as the first (so the button is not disabled) if it is the first AND there is no unlocked entries before it.
